test/simple.test.mo

```diff
@@ -12,26 +12,52 @@ func f(g : () -> async ()) : async () {
   Debug.print("after g");
 };
 
-let mock = AsyncTester.SimpleStageTester<()>(null);
+do {
+  let mock = AsyncTester.SimpleStageTester<()>(null);
 
-func g() : async () {
-  mock.call_result(await* mock.call());
-};
+  func g() : async () {
+    mock.call_result(await* mock.call());
+  };
 
-do {
-  let response = mock.stage(?());
-  let fut = f(g);
-  await async ();
-  Debug.print("waiting");
-  mock.release(response);
-  await fut;
+  do {
+    let response = mock.stage(?());
+    let fut = f(g);
+    await* mock.wait(0);
+    Debug.print("waiting");
+    mock.release(response);
+    await fut;
+  };
+
+  do {
+    let response = mock.stage(null);
+    let fut = f(g);
+    await* mock.wait(0);
```

Comment:
Maybe wait(response) because below we have release(response) and not release(0).

Reply:
wait(0) is wrong anyway, must be wait(1)